src/components/tabs/tab-bar.tsx

```diff
@@ -222,6 +222,9 @@ const TabBar = (props: TabBarProps) => {
         {tabList.map((child: any, i) => {
           const className = child.tabComponent.props.className;
           const activeClassName = child.tabComponent.props.classNameActive;
+          const closeableTab = child.tabComponent.props.closeable
```

Comment:
Why not use defaultProps?

Reply:
just missed it, definitely good to put it, in this case specifically,  the props are inside the child component not on the parent itself, I defined the variables here for more readability when using it on the jsx after